packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -79,7 +80,24 @@ export class AtomicBreadbox implements InitializableComponent {
 
   private breadcrumbShowLessFocus?: FocusTargetController;
 
+  /**
+   * This prop allows you to control the display of a path by
+   * limiting the number of visible values in the path.
```

Comment:
Maybe it's me but, `values` does not seem like the right term here. Can we say `directory` or `level`?

Reply:
Mh yeah this could be phrased a bit better maybe.
I think I would go with something like:
>  This prop allows you to control how many ancestor/parent values of the selected value to display.

Maybe @spiracha-coveo could chime in and give us some API Writer magic 🗡️

Reply:
👋 
Is this more clearer:
> This prop allows you to control the display depth of the path by specifying the number of parent or ancestor breadcrumb links relative to the currently selected value.